### PR TITLE
289 implement eth get storage at

### PR DESCRIPTION
**Pull request by ivandzen**
Implement eth_getStorageAt RPC method, add UT

### CODE REVIEW

**Review by vasiliy-zaznobin — DISMISSED** [review]

File: proxy/plugin/solana_rest_api.py [hunk 1]
```diff
@@ -195,6 +195,17 @@ def getBlockBySlot(self, slot, full):
         }
         return ret
 
+    def eth_getStorageAt(self, account, position, block_identifier):
+        '''Retrieves storage data by given position
+        Currently supports only 'latest' block
+        '''
+        if block_identifier != "latest":
+            print(f"Block type '{block_identifier}' is not supported yet")
+            raise RuntimeError("Not supported block")
```
Comment (s-medvedev) [inline proxy/plugin/solana_rest_api.py]: `Not supported block` seems confused. Please, change to `Not supported block identifier`.

File: proxy/plugin/solana_rest_api.py [hunk 2]
```diff
@@ -195,6 +195,17 @@ def getBlockBySlot(self, slot, full):
         }
         return ret
 
+    def eth_getStorageAt(self, account, position, block_identifier):
+        '''Retrieves storage data by given position
+        Currently supports only 'latest' block
+        '''
+        if block_identifier != "latest":
+            print(f"Block type '{block_identifier}' is not supported yet")
```
Comment (s-medvedev) [inline proxy/plugin/solana_rest_api.py]: Use `logger` for printing.

File: proxy/plugin/solana_rest_api.py [hunk 3]
```diff
@@ -195,6 +195,17 @@ def getBlockBySlot(self, slot, full):
...
+            raise RuntimeError("Not supported block")
+        value = neon_cli().call('get-ether-storage-at', account, position)
```
Comment (s-medvedev) [inline proxy/plugin/solana_rest_api.py]: What happens here in the case when `neon-cli get-ether-storage-at` return an error?

File: proxy/plugin/solana_rest_api.py (hunk 1 above)
Comment (ivandzen) [reply proxy/plugin/solana_rest_api.py]: fixed

File: proxy/plugin/solana_rest_api.py (hunk 2 above)
Comment (ivandzen) [reply proxy/plugin/solana_rest_api.py]: fixed

File: proxy/plugin/solana_rest_api.py (hunk 3 above)
Comment (ivandzen) [reply proxy/plugin/solana_rest_api.py]: This case occured when account not found in solana (or account is not contract account) and must be treated as access to non-existing storage slot and return zero

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**